### DISCUSSION TITLE
Tracking a team over multiple competitions

### DISCUSSION THREAD

**Discussion by brhahlen** [vasqued2/ha-teamtracker#87] [Q&A] [answered]
First: Thank you for this awesome integration and the card!

It looks like we need to create individual sensors for a team in a league to track.
The question I have, is whether it would be possible to track a Team over multiple leagues?
For instance: Manchester United plays in English Premier League, European League (either CL, Europa League or the other one), and then the two cups in England, which means 6 sensors.
Or the Dutch National Football team, who plays in European/World Cup Qualifiers, the actual European/World Championship, Friendlies and the Nations League.

So ideally, you would set up one sensor for the team, that tracks it over different leagues.
Is that at all possible? Or too technically challenging?

Thanks in advance for the answer!

**Comment by brhahlen** [comment]
@vasqued2 Sorry for the ping, is this something that would be possible?
Or is it way beyond the scope?

**Comment by vasqued2** [comment, marked as answer]
You have to set up a sensor for each league.  This is a limitation of the ESPN API.  It uses a different API call for each league.

The best way to get around this is to set up a sensor and card for each league and then hide the card when there is no upcoming game so you only see the card for the league with an upcoming game.  The Wiki shows 2 ways to do this.

[https://github.com/vasqued2/ha-teamtracker/wiki/Using-Auto-Entities-Card-to-Hide-NOT_FOUND-Status](https://github.com/vasqued2/ha-teamtracker/wiki/Using-Auto-Entities-Card-to-Hide-NOT_FOUND-Status)

[https://github.com/vasqued2/ha-teamtracker/wiki/Using-Conditional-Cards-to-Hide-NOT_FOUND-Status](https://github.com/vasqued2/ha-teamtracker/wiki/Using-Conditional-Cards-to-Hide-NOT_FOUND-Status)
